cargo-shuttle/src/lib.rs

```diff
@@ -439,12 +436,13 @@ impl Shuttle {
             }
         });
 
+        let service_name = self.ctx.project_name();
         let working_directory = self.ctx.working_directory();
 
         trace!("building project");
         println!(
-            "{:>12} {}",
-            "Building".bold().green(),
+            "{} {}",
+            "    Building".bold().green(),
```

Comment:
I might be a bit confused, why is this change needed?

Reply:
The bold and green modifiers add control characters before and after the word, meaning it is already more than 12 bytes/chars long when given to the formatter, meaning no spaces are added. (Try it yourself :D)

You can solve this by either adding the correct amount of space, or to add the space alignment before bold+green, resulting in 2 format-like calls.

I chose the former.

Reply:
Ahh, of course :man_facepalming: